libraries/botbuilder-storage-dynamodb/.gitignore

```diff
@@ -0,0 +1,63 @@
+# Logs
```

Comment:
Remove `.gitignore` since the monorepo covers this. If you have specific directories that the root `.gitignore` doesn't cover, you can keep that local.